python/tk_desktop/desktop_window.py

```diff
@@ -524,10 +524,10 @@ def add_to_project_menu(self, action):
         self.project_menu.insertAction(self.__pipeline_configuration_separator, action)
 
     def add_project_command(
-            self, name, button_name, menu_name, icon, command_tooltip, groups
-    ):
+            self, name, button_name, menu_name, icon, command_tooltip, groups, is_menu_default
+        ):
         self._project_command_model.add_command(
-            name, button_name, menu_name, icon, command_tooltip, groups
+            name, button_name, menu_name, icon, command_tooltip, groups, is_menu_default
```

Comment:
please add doc strings whenever you touch a method which doesn't have docstrings defined :)